vacatary/routes/test.js

```diff
@@ -0,0 +1,77 @@
+var express = require('express');
```

Comment:
duplicated file

Reply:
Don't understand this one.

Reply:
It seems that you have two identical files `vacatary/routes/test.js` and `backend/routes/test.js`